reana_client/cli/files.py

```diff
@@ -208,8 +215,18 @@ def download_files(ctx, workflow, filenames, output_directory, access_token):
 @add_workflow_option
 @add_access_token_options
 @click.pass_context
-def upload_files(ctx, workflow, filenames, access_token):
-    """Upload files and directories to workflow workspace."""
+def upload_files(ctx, workflow, filenames, access_token):  # noqa: D301
+    """Upload files and directories to workspace.
+
+    The `upload` command permits to upload workflow input files and
+    directories. The SOURCES can be specified as the command-line argument (see
```

Comment:
The SOURCES argument can be repeated and specifies which files and directories are to be uploaded, see examples below.